modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/ObjectEntryManagerImpl.java

```diff
@@ -364,6 +424,47 @@ private String _getObjectEntryPermissionName(long objectDefinitionId) {
 		return ObjectDefinition.class.getName() + "#" + objectDefinitionId;
 	}
 
+	private boolean _hasRelatedObjectEntries(
+			com.liferay.object.model.ObjectEntry objectEntry)
+		throws PortalException {
+
+		for (ObjectRelationship objectRelationship :
+				_objectRelationshipsWithPreventDeletionType) {
+
+			int count = 0;
+
+			if (StringUtil.equals(
+					objectRelationship.getType(),
+					ObjectRelationshipConstants.TYPE_ONE_TO_MANY)) {
+
+				count =
+					_objectEntryLocalService.
+						getOneToManyRelatedObjectEntriesCount(
+							objectEntry.getGroupId(),
+							objectRelationship.getObjectRelationshipId(),
+							objectEntry.getPrimaryKey());
+			}
+
+			if (StringUtil.equals(
+					objectRelationship.getType(),
+					ObjectRelationshipConstants.TYPE_MANY_TO_MANY)) {
+
+				count =
+					_objectEntryLocalService.
+						getManyToManyRelatedObjectEntriesCount(
+							objectEntry.getGroupId(),
+							objectRelationship.getObjectRelationshipId(),
+							objectEntry.getPrimaryKey(), true);
+			}
+
+			if (count > 0) {
+				return true;
+			}
```

Comment:
@p-albuquerque, I believe you can return the method result without using an auxiliary variable.
Java accepts numbers as boolean.
false = 0 (zero)
true = different from 0 (zero)